org.freedesktop.Sdk.Extension.tup.yaml

```diff
@@ -0,0 +1,28 @@
+id: org.freedesktop.Sdk.Extension.tup
+branch: '24.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+command: tup
+modules:
+  - name: tup
+    sources:
+      - type: archive
+        url: https://gittup.org/tup/releases/tup-v0.8.tar.gz
+        sha512: 3f09a3defc97d174ed85d84679ad4a41bd83e12d84fa151a38c2aedd613f2739989fd9ebeb2d316bd716cf5afd3aead64d7d0cad968e705fe4092c5e7fd9a0ee
+      - type: patch
+        path: tup-disable-unconditional-x86-arch-siblings.patch
+    buildsystem: simple
+    build-options:
+      env:
+        TUP_SERVER: ldpreload
+    build-commands:
+      - echo CONFIG_TUP_USE_SYSTEM_PCRE=y >> ./tup.config
+      - echo CONFIG_TUP_USE_SYSTEM_SQLITE=y >> ./tup.config
+      - echo CONFIG_TUP_SERVER=ldpreload >> ./tup.config
+      - echo CONFIG_X86_ARCH_SIBLINGS=n >> ./tup.config
```

Comment:
is there no better way to do this? echo-ing things should not be part for a proper buildsystem.